js/script.js

```diff
@@ -0,0 +1,20 @@
+﻿function Phone(brand, price, color) {
+	this.brand = brand; 
+	this.price = price;
+	this.color = color;
+}
+Phone.prototype.printInfo = function() {
+		console.log("Brand of the phone is " + this.brand + ", color is " + this.color + ", and the price is " + this.price + "pln.");
+}
+
+var iPhone6S = new Phone("Apple", 2250, "silver");
```

Comment:
move declaration of your variables to top of current scope :) it is a good practise